docs/source/continuous-deployment.rst

```diff
@@ -0,0 +1,26 @@
+Continuous Deployment
+======================
+We utilize OpenShift to deploy our Sync2Jira instance. Here we assume you have configured your quay.io repo to push to some sort of message bus. This will likely have to be done with an external script (something we do not support).
```

Comment:
You can link the reader to https://github.com/release-engineering/repotracker which is what we use to publish on the bus when quay.io images update.

Reply:
Good idea